docs/src/Callout.doc.js

```diff
@@ -22,168 +47,354 @@ card(
+            }}
+          />
+        `}
+      />
+
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        - Stack Callouts. In the case that banners must be stacked, Callouts will usually come before Upsells.
+        `}
+        defaultCode={`
+        <Box>
+          <Box marginBottom={4} display="flex" alignItems="center">
+            <Icon icon="pinterest" color="red" size={32}/>
+            <ButtonGroup>
+              <Button color="transparent" iconEnd="arrow-down" text="Business" inline />
+              <Button color="transparent" iconEnd="arrow-down" text="Create" inline />
+              <Button color="transparent" iconEnd="arrow-down" text="Analytics" inline />
+              <Button color="transparent" iconEnd="arrow-down" text="Ads" inline />
+            </ButtonGroup>
+          </Box>
+          <Divider/>
+          <Box marginTop={8}>
+            <Flex gap={2} direction="column">
+              <Upsell
+                imageData={{
+                  component: <Icon icon="send" accessibilityLabel="Send" color="darkGray" size={32}/>
+                }}
+                title="So close! Finish installing your pinterest tag, get $10 in ads credit"
+                message="Track ads conversion—sales, traffic and more—with the Pinterest tag"
+                primaryAction={{label: "Claim now", accessibilityLabel:""}}
+              />
+              <Callout
+                type="info"
+                iconAccessibilityLabel="Info"
+                title="We have not yet detected your tag"
+                message="It may take up to 10 minutes to automatically detect a newly installed tag. If you'd like to manually verify your tag, please click the Verify Tag button."
+                primaryAction={{label: "Verify Tag", accessibilityLabel:""}}
+                dismissButton={{
+                  accessibilityLabel: 'Dismiss this banner',
+                  onDismiss: ()=>{},
+                }}
+              />
+            </Flex>
+          </Box>
+          <Box height={150}/>
+        </Box>
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      The \`iconAccessibilityLabel\`, \`dismissButton\`, \`primaryAction\`, and \`secondaryAction\` each require a short, descriptive label for screen readers. These labels should communicate intent or visually describe what any icons might mean, such as “error”, “info” or “warning”. They should also be localized.
+
+      In the case of [Buttons](/Button), alternative text should be provided to replace vague text like "Visit" or "Learn more" with more descriptive information, like ‘Learn more about our work from home resources’.
+
+      For [IconButtons](/IconButtons), like the \`dismissButton\`, the label provided should indicate the intent, like “Dismiss this banner”.
```

Comment:
This wording makes it sound like we're providing `dismissButton` as an example of IconButton (which it is, but that's irrelevant). I think what we're going for here is more "similar to `dismissButton`…"